.gitignore

```diff
@@ -74,3 +74,4 @@ tags
 .DS_Store
 .idea
 *.sublime-workspace
+version.txt
```

Comment:
You should be generating this file in the build dir not here. So it shouldn't be git ignored.